src/segments/Footer.tsx

```diff
@@ -47,90 +73,148 @@ function Foot() {
 
       // POST request using fetch inside useEffect React hook
       const requestOptions = {
-          method: 'POST',
-          mode: 'no-cors',
-          headers: {
-            'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'
-          },
-          body: formBody
+        method: "POST",
+        mode: "no-cors",
+        headers: {
+          "Content-Type": "application/x-www-form-urlencoded;charset=UTF-8",
+        },
+        body: formBody,
       };
 
-      fetch('https://tools.epns.io/sendy/subscribe', {
-          method: 'POST',
-          mode: 'cors',
-          headers: {
-            'Content-Type': 'application/x-www-form-urlencoded;charset=UTF-8'
-          },
-          body: formBody
+      fetch("https://tools.epns.io/sendy/subscribe", {
+        method: "POST",
+        mode: "cors",
+        headers: {
+          "Content-Type": "application/x-www-form-urlencoded;charset=UTF-8",
+        },
+        body: formBody,
+      })
+        .then((response) => response.json())
+        .then((jsondata) => {
+          // console.log(jsondata);
+          setMailListProcessing(2);
         })
-        .then(response => response.json())
-        .then(jsondata => {
-            // console.log(jsondata);
-            setMailListProcessing(2);
-          })
-        .catch(err => {
+        .catch((err) => {
           // console.log(err);
-          setMailListError("Mayday! Mayday! something went wrong. Please retry...");
+          setMailListError(
+            "Mayday! Mayday! something went wrong. Please retry..."
+          );
           setMailListProcessing(0);
         });
-    }
-    else {
+    } else {
       setMailListError("Incorrect e-mail, please check and retry!");
       setMailListProcessing(0);
     }
-  }
+  };
   // ---------
 
   // HELPER METHODS
   // ---------
   const validateEmailFooter = (email) => {
-    const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    const re =
```

Comment:
Better to give meaningful name for this regex

Reply:
The naming convention has been already given before. So I am not sure whether we should change this or not.